sofy_sofy/README.md

```diff
@@ -0,0 +1,45 @@
+# Sofy
+
+## Overview
+
+### What is Sofy 
+SOFY is a no-code test automation platform that offers end-to-end testing from the CI/CD pipeline on real devices, making the testing process faster, more efficient, and more accessible than ever before.
```

Comment:
```suggestion
SOFY is a no-code test automation platform that offers end-to-end testing from the CI/CD pipeline on real devices, making the testing process faster, more efficient, and more accessible than ever before.
```

Reply:
We try to avoid marketing material for integrations in integrations-extras. Can you update this to include what SOFY does in 1-2 sentences?